### ISSUE TITLE
Custom ITask with Output of Reference is not showed in the Projects Dependencies

### ISSUE THREAD

**Issue by hknielsen** [dotnet/sdk#28694]

**Comment by kvenkatrajan** [comment]
@marcpopMSFT can you have a look please?

**Comment by rainersigwald** [comment]
@kvenkatrajan I agree with @hknielsen that this is likely a project-system issue. Specifically I suspect what's needed is documentation and guidance: "How should a reference-generating target behave to properly work in design-time builds, so that added references appear in the solution explorer tree?"
